src/basic/headers-util.c

```diff
@@ -0,0 +1,235 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
```

Comment:
i don't thing headers-util.[ch] should be in basic/. AFAICS it's only useful within the journal-remote context, and hence should be moved there